examples/tutorial/13_Deploying_Bokeh_Apps.ipynb

```diff
@@ -101,13 +101,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import holoviews as hv, geoviews as gv, param, dask.dataframe as dd, panel as pn\n",
+    "import holoviews as hv\n",
+    "import geoviews as gv\n",
+    "import param\n",
+    "import dask.dataframe as dd\n",
+    "import panel as pn\n",
```

Comment:
In Python source code that people read with an editor and only read when they actually want to examine the code itself, I think PEP8 applies, but I don't think that's actually good practice for notebooks.  Notebooks should be readable, with the Markdown text, the image outputs, and the meaningful code dominating the page; a long imports list is just a distraction that I'd prefer to get over with as quickly as possible.  So I don't prefer expanding them out in user examples like this.

Reply:
ok makes sense

Reply:
(Or at least notebooks used for communication, as examples.  Notebooks written by one person for their own use can be done however the author likes, but I'm arguing that PEP8 import style isn't the right approach for a narrative notebook being shared with others.)